### PR TITLE
Nzpmc 015/setup router

### PR DESCRIPTION
**Pull request by matthewaptaylor**
Issue
Setup router, a single universal font and background

Risks
Builds properly, no known risks

### CODE REVIEW

**Review by Will-ZYS — DISMISSED** [review]
Looks great! Thanks Matthew

**Comment by matthewaptaylor** [comment]
Sorry - forgot to update the favicon and realised after you'd already approved the PR. Everything's okay now. No known risks.

File: vue.config.js [hunk 1]
```diff
@@ -1,12 +1,3 @@
...
 }
-
-module.exports = {
-    chainWebpack: (config) => {
-        config.plugin("html").tap((args) => {
-            args[0].title = "NZPMC Online";
-            return args;
-        });
-    },
-};
```
Comment (Will-ZYS) [inline vue.config.js]: Isn't it just because this is not formated using prettier?

**Review by scorpionknifes — APPROVED** [review]
LGTM

**Review by Will-ZYS — APPROVED** [review]

**Pull request merged**